tests/data/token_indexers/elmo_indexer_test.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=no-self-use,invalid-name,bad-continuation,bad-whitespace
```

Comment:
Can you fix the bad-continuation and bad-whitespace, rather than overriding them here?

Reply:
Probably not without wanting to tear my eyes out or using a code formatter.  It's coming from the cut and pasted arrays of numbers in the tests.  Is there one for pylint?

Reply:
It's not that hard - you can use `bash scripts/pylint.sh` to run the pylint tests which get run on Travis.

Reply:
Is there a mode in pylint that will automatically reformat whitespace? Or another program that will do so according to our pylint configuration since we aren't following e.g. pep8 or another established style guide?  Otherwise I'd rather not spend 15 minutes making whitespace changes... hence the comment about tearing my eyes out.

Reply:
E.g. `yapf -i --style=pep8 tests/data/token_indexers/elmo_indexer_test.py` formats according to pep8 but fails pylint horribly.

Reply:
Please format these arrays properly and remove the global pylint statements at the top. I promise it will take you less than 10 minutes. You need to format arrays to look like this:
```
array = [[0, 0, 0, 1],
         [1, 2, 3, 4]]

```
I.e 1 space indentation for each dimension, one space between each number.